content/en/docs/concepts/storage/volumes.md

```diff
@@ -574,10 +576,17 @@ backoff and will be reported on the pod reason and message.
 
 The types of objects that may be mounted by this volume are defined by the
 container runtime implementation on a host machine and at minimum must include
-all valid types supported by the container image field. The OCI object gets
-mounted in a single directory (`spec.containers[*].volumeMounts.mountPath`) by
-will be mounted read-only. On Linux, the container runtime typically also mounts the 
-volume with file execution blocked (`noexec`).
+all valid types supported by the container image field. 
+
```

Comment:
Maybe:

```suggestion

#### `image` volume limitations

```
and at near the end of the text being added, another heading such as

```markdown
#### Fields for `image` volumes {#volume-image-fields}
```

```

Reply:
Fixed as suggested.